src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.img__link {
+  display: block;
+  height: 40px;
+}
+
+header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.nav {
+  display: block;
```

Comment:
I'm not convinced if it is necessary. Does anything break when you remove it?

Reply:
yeah, it's not necessary.